governance/lead-proposal-process.md

```diff
@@ -0,0 +1,153 @@
+# TAG Leadership election process
+
+The leadership team of the TAG (Chairs and TLs) as well as the extended leadership team of working group chairs and project leads are servant leads in the community.
+They are an enabler for the group and support the creation and success of the different activities.
+They are the driver for the group for not losing the higher targets out of sight and taking decisions when no consensus can be found.
+
+Roles within the TAG outlining expectations and responsibilities are defined in the [roles](roles.md) governance document.
+
+The election process for TAG Leadership is described in the first part of this document.
+The election process for Working Group and Project leadership is more lightweight and defined in the second part of this document.
+
+- [TAG Leadership election process](#tag-leadership-election-process)
+  - [TAG Chair and TL election process](#tag-chair-and-tl-election-process)
+    - [Process of nominations](#process-of-nominations)
+      - [Community timeline](#community-timeline)
+      - [Submitting a nomination](#submitting-a-nomination)
+      - [Additional note about the TAG Chair proposal process](#additional-note-about-the-tag-chair-proposal-process)
+    - [General nomination requirements](#general-nomination-requirements)
+      - [Justification for nominees](#justification-for-nominees)
+    - [Announcing nomination results](#announcing-nomination-results)
+    - [Next Steps](#next-steps)
+  - [Working Group and Project Leadership](#working-group-and-project-leadership)
+    - [Process of nominations](#process-of-nominations-1)
+      - [Community timeline](#community-timeline-1)
+      - [Submitting a nomination](#submitting-a-nomination-1)
+
+## TAG Chair and TL election process
+
+### Process of nominations
+
+TAG Lead roles are open to all interested participants within the community. 
+Depending on the needs of the community, as determined by the current TAG leadership team (Chairs and TLs), nominations may come from working group co-chairs, TAG co-chairs and/or tech leads and/or other community members. Each TAG is expected to define this for themselves.
+
+The TAG ENV leadership team is conducted of:
+
+* Three Co-Chairs
+* Tech Leads
+
+Final nominations will adhere to the [CNCF's TOC described process](https://github.com/cncf/toc/blob/main/tags/cncf-tags.md#elections) as well as to the leadership nomination requirements described below.
+
+Any individual who is a member of the TAG ENV, it's working group's, or otherwise considered an active participant of the group may nominate another individual.
+
+The process of endorsing and selecting individuals will consider the following factors to ensure both the need for proper community representation and the necessary technical expertise:
+
+* Gender diversity
```

Comment:
I'd order this differently, and start with "active engagement in the community", "length of engagement in the community", and "existing body of work."

Reply:
agreed